rewrite-test/src/main/kotlin/org/openrewrite/java/cleanup/UselessCompoundTest.kt

```diff
@@ -0,0 +1,113 @@
+package org.openrewrite.java.cleanup
+
+import org.junit.jupiter.api.Test
+import org.openrewrite.Recipe
+import org.openrewrite.java.JavaParser
+import org.openrewrite.java.JavaRecipeTest
+
+interface UselessCompoundTest : JavaRecipeTest {
+    override val recipe: Recipe
+        get() = UselessCompound()
+
+    @Test
+    fun removeUselessCompoundAnd(jp: JavaParser) = assertChanged(
```

Comment:
Similar to previous suggestions, I'd rename this to something like simplifyCompoundAnd.